lib/content-services/src/lib/search/components/search-facet-field/search-facet-field.component.ts

```diff
@@ -61,7 +61,7 @@ export class SearchFacetFieldComponent implements FacetWidget {
     selectFacetBucket(field: FacetField, bucket: FacetFieldBucket) {
         if (bucket) {
             bucket.checked = true;
-            this.queryBuilder.addUserFacetBucket(field, bucket);
+            this.queryBuilder.addUserFacetBucket(field.field, bucket);
```

Comment:
Same as unit tests, field.field sounds weird

Reply:
Renamed

Reply:
I still see instances of field.field in this file, along with some others in 

- search-filter-chips.component.html
- search-query-builder.service.spect.ts

These 2 aren't introduced by your changes though. These are pre-existing